infrastructure/general/setup-secure-wss.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Setup Secure WebSocket
+description: Instructions on setting up a secure socket for remote connections.
+---
+
+<!-- TODO: link relevant guides (bootnode, rpc) -->
+
+## Secure a WS Port
+
+A non-secure ws port can be converted to a secure WSS port by placing it behind an SSL-enabled proxy. This can be used to secure a bootnode or secure a RPC server. The SSL-enabled apache2/nginx/other proxy server redirects requests to the internal WebSocket and converts it to a secure (WSS) connection. For this, you will need an SSL certificate for which you can use a service like LetsEncrypt or self-signing.
```

Comment:
```suggestion
A non-secure WS port can be converted to a secure WSS port by placing it behind an SSL-enabled proxy. This can be used to secure a bootnode or secure a RPC server. The SSL-enabled apache2/nginx/other proxy server redirects requests to the internal WebSocket and converts it to a secure (WSS) connection. For this, you will need an SSL certificate for which you can use a service like LetsEncrypt or self-signing.
```